packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -337,8 +344,9 @@ export default class ComboBox extends React.Component {
             </ListBox.Field>
             {isOpen && (
               <ListBox.Menu aria-label={ariaLabel} id={id}>
-                {this.filterItems(items, itemToString, inputValue).map(
-                  (item, index) => (
+                {this.filterItems(items, itemToString, inputValue)
+                  .sort(dropdownSort)
```

Comment:
the `dropdownSort` prop is optional but it looks like it's always being called, so should we assign it a default value?

Reply:
I misread the documentation for `Array.prototype.sort` -- you are correct, we should provide a default value there. Otherwise, it will attempt to convert the items to strings and then sort them. 

The desired default functionality would be to preserve the order of elements, correct?

Reply:
yeah I think we can probably just pass in an empty function

Reply:
I am not sure that an empty function would preserve the order, as that would evaluate to `undefined` and the `sort` function would use that value.  I defaulted it to a function that always returns 0.

Reply:
passing in an empty function wouldn't be the same as passing in `undefined`, so wouldn't `.sort(() => {})` preserve the order?